README.md

```
@@ -103,15 +103,18 @@ This script handles all dependencies and configuration automatically for a seaml
 
 ### Option 3: Installing via Smithery ⭐ **Auto-Updates** **Requires Node.js**
 
-To install Desktop Commander for Claude Desktop automatically via [Smithery](https://smithery.ai/server/@wonderwhy-er/desktop-commander):
+To install Desktop Commander for Claude Desktop via [Smithery](https://smithery.ai/server/@wonderwhy-er/desktop-commander):
 
-```bash
-npx -y @smithery/cli install @wonderwhy-er/desktop-commander --client claude
-```
+1. **Visit the Smithery page:** https://smithery.ai/server/@wonderwhy-er/desktop-commander
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)